### PR TITLE
Polyhedron_demo : Use floats in the Point_with_normals_item

### LINKED ISSUE #1754: 3D demo (Polyhedron_3) crashes while trying to open a large .ply file.
Versions: Using CGAL 4.9 64 bits, Polyhedron_3 
OS: Fedora 24 x86_64 
Processor: Intel Core i7-6820HQ CPU @ 2.70GHz x4 
Memory: 32 GB 
Graphics Card: Nvidia GM107GLM Quadro M2000M with 4GB of memory

Bug Description: When Polyhedron_3 is loading a very large point set with color attributes, from a .ply file (3.4Gb; 61,020,661 vertices) it crashes after 6 minutes,  and closes - nothing is rendered on the screen. However the name of the file is displayed on the item list. 

Note: Cédric Portaneri (newly recruited engineer at Inria) reported the bug and can provide you with the file. You can contact him at cedric.portaneri@inria.fr

Steps to Reproduce: 
	1. Go to File > Load...
	2. Open C3DC_StatueZ2-Clean.ply

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR fixes #1754 . We use floats instead of doubles in the Scene_point_with_normals_item to reduce the needed memory.

 This is a temporary fix, as we only gain a factor 2 in the acceptable memory size. A better fix could be to split the data into several VBOs.

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp [hunk 1]
```diff
@@ -735,7 +733,7 @@ void Scene_points_with_normal_item::computes_local_spacing(int k)
             );
   Distance tr_dist(d->m_points->point_map());
 
-  if (!(d->m_points->has_property_map<double> ("radius")))
+  if (!(d->m_points->has_property_map<CGAL_data_type> ("radius")))
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp]: I am pretty sure that change is not wanted.

**Comment by lrineau** [comment]
This PR was tested in CGAL-4.10-Ic-86 successfully. I am waiting for the manual testing by one of us, particularly on Windows.

**Comment by sgiraudot** [comment]
Tested locally (GNU/Linux), it works well.

**Comment by afabri** [comment]
Tested locally (VC++, Win10), it works well.

**Pull request merged**